vmdb/.gitignore

```diff
@@ -33,6 +33,72 @@ config/database.yml*
 config/vmdb.yml.db
 config/miq_pass
 
+# bin/
+bin/autospec
+bin/aws-rb
+bin/brakeman
+bin/bundler
+bin/byebug
+bin/churn
+bin/coderay
+bin/dot2ruby
+bin/erubis
+bin/fission
+bin/flay
+bin/flog
+bin/fog
+bin/gem2gv
+bin/git2gv
+bin/haml
+bin/htmldiff
+bin/httpclient
+bin/jasmine
+bin/ldiff
+bin/lessc
+bin/nokogiri
+bin/oauth
+bin/pry
+bin/r2r_show
+bin/rackup
+bin/rails_best_practices
+bin/rdoc
+bin/reek
+bin/restclient
+bin/ri
+bin/rmsgcat
+bin/rmsgfmt
+bin/rmsginit
+bin/rmsgmerge
+bin/roodi
+bin/roodi-describe
+bin/rspec
+bin/ruby-parse
+bin/ruby-prof
+bin/ruby-prof-check-trace
+bin/ruby-rewrite
+bin/ruby2gv
+bin/ruby_parse
+bin/ruby_parse_extract_error
+bin/rubyrep
+bin/rxgettext
+bin/safe_yaml
+bin/saikuro
+bin/sass
+bin/sass-convert
+bin/scss
+bin/slimrb
+bin/sprockets
+bin/techbook
+bin/thin
+bin/thor
+bin/tilt
+bin/tt
+bin/unparser
+bin/wsdl2ruby.rb
+bin/xml2gv
+bin/xsd2ruby.rb
+bin/ziyafy
+
```

Comment:
You don't have to be so explicit.

You can just put

```ruby
bin/*
```

gitignore only watches untracked files, so this would hide the untracked files, while still allowing changes to the tracked files (e.g. bin/rails)